WordPress/Classes/ViewRelated/System/Floating Create Button/FloatingActionButton.swift

```diff
@@ -30,6 +29,7 @@ class FloatingActionButton: UIButton {
         super.draw(rect)
 
         layer.cornerRadius = rect.size.width / 2
+        layer.backgroundColor = UIColor(light: .label, dark: .systemGray2).cgColor
```

Comment:
@osullivanchris 👋 
Ended up using system colors for this - label for light, system gray 2 for dark

Reply:
@kean I'm setting the background color in draw() because CGColor doesn't adapt to system appearance changes

https://developer.apple.com/documentation/uikit/appearance_customization/supporting_dark_mode_in_your_interface